packages/scanner/zxing/android/build.gradle

```diff
@@ -0,0 +1,50 @@
+group 'com.example.scanner_zxing'
```

Comment:
Do we want to change this?

Reply:
I'm not sure this is relevant, as this dependency will always be independant.